### PR TITLE
Add documentation how to run a node from snapshot

### PR DESCRIPTION
**Pull request by IvayloPapazov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,28 @@ docker run -p 3013:3013 -p 3015:3015 \
     aeternity/aeternity
 ```
 
+#### Restore from snapshot
+
+To speedup the initial blockchain syncronization the node database can be restored from a snapshot following the below steps:
```
Comment (venimus) [inline README.md]: synchronization

File: README.md [hunk 2]
```diff
@@ -62,6 +62,28 @@ docker run -p 3013:3013 -p 3015:3015 \
...
+
+* delete the contents of the database if the node has been started already
+* download the database snapshot
+* verify if the snapshot checksum match the downloaded file
+* unarchive the database snapshot
+
+**Note that the docker container must be stopped before replacing the database**
+
+The following snipped can be used to delete current database and replace it the latest mainnet database snapshot and restore it assuming the database path is ` ~/.aeternity/maindb`:
```
Comment (venimus) [inline README.md]:
```suggestion
The following snippet can be used to replace the current database with the latest mainnet snapshot assuming the database path is ` ~/.aeternity/maindb`:
```

Comment (IvayloPapazov) [reply README.md]: 92ff9b7

File: README.md (hunk 1 above)
Comment (IvayloPapazov) [reply README.md]: 92ff9b7

**Pull request merged**